test/import/import-crd.test.ts

```diff
@@ -219,3 +237,120 @@ describe('classPrefix can be used to add a prefix to all construct class names',
     classNamePrefix: 'Foo',
   });
 });
+
+describe('safe parsing', () => {
+
+  test('description is sanitized', async () => {
+
+    const crd: ManifestObjectDefinition = {
+      apiVersion: 'apiextensions.k8s.io/v1beta1',
+      kind: 'CustomResourceDefinition',
+      metadata: {
+        name: 'testMetadata',
+      },
+      spec: {
+        group: 'testGroup',
+        names: {
+          kind: 'testNameKind',
+        },
+        validation: {
+          openAPIV3Schema: {
+            description: "*/console.log('hello')/*",
+          },
+        },
+      },
+    };
+
+    await withTempFixture(crd, async (fixture: string) => {
+      const definitions = await ImportCustomResourceDefinition.match({ source: fixture });
+      expect(definitions?.length).toEqual(1);
+      const definition = definitions ? definitions[0] : undefined;
```

Comment:
If we know the definitions length can we just access it directly?
```suggestion
      const definition = definitions[0];
```

Reply:
https://github.com/cdk8s-team/cdk8s-cli/pull/67#discussion_r709107115